### ISSUE TITLE
P1682R3 `to_underlying()` For Enumerations

### ISSUE THREAD

**Issue by StephanTLavavej** [microsoft/STL#1681]
[P1682R3](https://wg21.link/P1682R3) `to_underlying()` For Enumerations

Feature-test macro:
`#define __cpp_lib_to_underlying 202102L`

**Comment by SuperWig** [comment]
R3 is actually the latest revision and that just has a note to replace the value of the macro.

**Comment by CaseyCarter** [comment]
> R3 is actually the latest revision and that just has a note to replace the value of the macro.

It's a shame that WG21 didn't vote to merge R3 into the Working Draft =) I've communicated the SNAFU to the LWG Chair, who is abashed at having listed the incorrect revision in the motion to merge. LWG Chair is satisfied that the differences between R2 and R3 are editorial, and will ensure that the editors apply those differences as an editorial clarification after incorporating P1682R2 into the Working Draft.

TLDR: we should simply work to [P1682R3](https://wg21.link/p1682r3).

**Comment by CaseyCarter** [comment]
> > R3 is actually the latest revision and that just has a note to replace the value of the macro.
> 
> It's a shame that WG21 didn't vote to merge R3 into the Working Draft =) I've communicated the mixup to the LWG Chair, who is abashed at having listed the incorrect revision in the motion to merge. LWG Chair is satisfied that the differences between R2 and R3 are editorial, and will ensure that the editors apply those differences as an editorial wording clarification after incorporating P1682R2 into the Working Draft.
> 
> TLDR: we should simply work to [P1682R3](https://wg21.link/p1682r3).

**Comment by StephanTLavavej** [comment]
Thanks, I've edited this issue (and our internal spreadsheet) accordingly!
